bundle-size/app.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright 2018, the AMP HTML authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const {dbConnect} = require('./db');
+const path = require('path');
+
+const EMPTY_SHA = '0'.repeat(40);
+
+async function getBuildArtifactsFile(github, filename) {
+  if (filename == 'OWNERS') {
+    // Temporarily, until we add the OWNERS file to the build artifacts repo.
+    return 'choumx\ndanielrozenberg\njridgewell\nrsimha\n';
```

Comment:
Curious: Why not create an owners file right away and read from it?

Reply:
Will do and remove this line before submitting this PR